src/ddmd/traits.d

```diff
@@ -911,6 +912,61 @@ extern (C++) Expression semanticTraits(TraitsExp e, Scope* sc)
         auto tup = new TupleExp(e.loc, mods);
         return tup.semantic(sc);
     }
+    if (e.ident == Id.getFunctionVariadicStyle)
+    {
+        /* Accept a symbol or a type. Returns one of the following:
+         *  "none"      not a variadic function
+         *  "D"         extern(D) void dstyle(...), with __argptr and __arguments
+         *  "C"         extern(C) void cstyle(int, ...), use core.stdc.stdarg
+         *  "typesafe"  void typesafe(T[] ...)
+         */
+        // get symbol linkage as a string
+        if (dim != 1)
+            return dimError(1);
+
+        LINK link;
+        int varargs;
```

Comment:
That's from C macros and functions, __VA_ARGS__ and va_arg, but in the dmd frontend it's called varargs everywhere.